internal/app/networkd/pkg/nic/nic.go

```diff
@@ -408,7 +403,4 @@ func (n *NetworkInterface) Reset() {
 			continue
 		}
 	}
-
-	// nolint: errcheck
```

Comment:
I had to remove that, as due to the way boot phase works:

1. initial network setup - all is good, interfaces are up 
2. reset network step (which calls this function) filters all 'up' interfaces and brings them down by taking away `IFF_UP`
3. setup network step sees all interfaces are down, doesn't do anything and node tries to boot without networking

I'm not sure how it did work before my changes.

Probably we want this link down? If we do, we should do link up as part of reset sequence